docker-compose.yml

```diff
@@ -28,8 +28,6 @@ services:
       # Tracy
       - "8086:8086"
     entrypoint: cargo watch -i flamegraphs -i log.conf --why -C crates/standalone -x 'run start --data-dir=/stdb/data --jwt-pub-key-path=/etc/spacetimedb/id_ecdsa.pub --jwt-priv-key-path=/etc/spacetimedb/id_ecdsa'
-    healthcheck:
-      test: curl -f http://localhost/ping || exit 1
```

Comment:
I'm not sure what else uses our `docker-compose.yml` file so this change *could* potentially break something. I don't think it's a huge concern but why did we remove this health check?